crowbar_framework/app/views/installer/index.html.haml

```diff
@@ -0,0 +1,20 @@
+.col-lg-6.col-md-6.col-sm-6.col-xs-12
+  .panel.panel-default
+    .panel-heading
+      %h2
+        = t(".installation_header").html_safe
+        .btn-group.pull-right
+          = button_to t(".install_button"), install_path, class: "btn btn-success", method: :post
```

Comment:
As the action is called by a button/link without any form I would suggest to make it simply a GET route.